### DISCUSSION TITLE
add AboutCommand for Nova

### DISCUSSION THREAD

**Discussion by niekdemelker** [laravel/nova-issues#6319] [Ideas & Feature Requests]
I think it would be a nice addition if Nova registers itself in the `Artisan About` command. Pulse already does this and i like the extra info info. 

I already made something i placed in my NovaServiceProvider.

```php
AboutCommand::add('Nova', fn () => [
    'Version' => Nova::version(),
    'License Key' => Nova::checkLicenseValidity()
        ? '<fg=green;options=bold>VALID</>'
        : '<fg=red;options=bold>INVALID</>',
]);
``` 

The above will tell the version number and if your license is valid. There is probably more to show here, but this seems like a nice start.

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
